src/Angor/Avalonia/AngorApp/Features/Invest/Amount/Breakdown.cs

```diff
@@ -0,0 +1,13 @@
+namespace AngorApp.Features.Invest.Amount;
+
+public record Breakdown(
+    int Index,
+    long Amount,
+    double Weight,
```

Comment:
What is the meaning of weight? the percentage of the total?

Reply:
Good question. Since it goes from 0 to 1, it's not a percentage, but a proportion. I was wondering about a good name initially, and used that. But it's clearly misleading.

I think "Ratio" or "Proportion" would be clearer names. What do you think? Maybe "RatioOfTotal" makes it.

Reply:
I've used `RatioOfTotal`. Please, tell me if it's OK :)